src/coreclr/vm/callingconvention.h

```diff
@@ -1522,6 +1684,137 @@ int ArgIteratorTemplate<ARGITERATOR_BASE>::GetNextOffset()
 
     int argOfs = TransitionBlock::GetOffsetOfArgs() + m_ofsStack;
     m_ofsStack += cbArg;
+    return argOfs;
+#elif defined(TARGET_LOONGARCH64)
+
+    int cFPRegs = 0;
+    int flags = 0;
+
+    switch (argType)
+    {
+
+    case ELEMENT_TYPE_R4:
+        // 32-bit floating point argument.
+        cFPRegs = 1;
+        break;
+
+    case ELEMENT_TYPE_R8:
+        // 64-bit floating point argument.
+        cFPRegs = 1;
+        break;
+
+    case ELEMENT_TYPE_VALUETYPE:
+    {
+        // Handle HFAs: packed structures of 2-4 floats or doubles that are passed in FP argument
+        // registers if possible. NOT support on LOONGARCH.
+        _ASSERTE(!thValueType.IsHFA() && "unimplemented on LOONGARCH yet! type-HFA");
+
+        // Composite greater than 16bytes should be passed by reference
+        if (argSize > ENREGISTERED_PARAMTYPE_MAXSIZE)
+        {
+            argSize = sizeof(TADDR);
+        }
+        else
+        {
+            MethodTable* methodTablePtr = nullptr;
```

Comment:
A nit - the convention used for pointers in the runtime is to start the variable with `p`, so the preferred name for this local would be `pMethodTable`.

Reply:
> A nit - the convention used for pointers in the runtime is to start the variable with `p`, so the preferred name for this local would be `pMethodTable`.

Thanks for your advice and I will rename it.